soh/soh/Enhancements/randomizer/option_descriptions.cpp

```diff
@@ -41,6 +41,7 @@ void Settings::CreateOptionDescriptions() {
                                                   "\n"
                                                   "Open - Sleeping Waterfall is always open. "
                                                   "Link may always enter Zora's Domain.";
+    mOptionDescriptions[RSK_LOCK_OVERWORLD_DOORS] = "Add locks to all overworld wooden doors, requiring specific small keys to open them";
```

Comment:
If my english isn't shite, I think it's not `Add locks to all overworld wooden doors` but `Add locks to all wooden overworld doors`

Reply:
```suggestion
    mOptionDescriptions[RSK_LOCK_OVERWORLD_DOORS] = "Add locks to all wooden overworld doors, requiring specific small keys to open them";
```